test/wallet-test.js

```diff
@@ -1912,8 +1909,107 @@ describe('Wallet', function() {
     }
   });
 
-  it('should cleanup', async () => {
-    consensus.COINBASE_MATURITY = 100;
-    // await wdb.close();
+  it('should count pending ancestors', async () => {
+    // Create wallet and get one address
+    const wallet = await wdb.create();
+    const addr1 = await wallet.receiveAddress();
+
+    // Dummy address for outputs
+    const recAddr = Address.fromHash(Buffer.alloc(20, 1));
+
+    // Add one single, unconfirmed coin to wallet
+    const mtx = new MTX();
+    mtx.addInput(dummyInput());
+    mtx.addOutput(addr1, 10 * 1e8);
+    const tx0 = mtx.toTX();
+    await wallet.txdb.add(tx0, null);
+
+    let ancs = null;
+    ancs = await wallet.getPendingAncestors(tx0);
+    assert.strictEqual(ancs.size, 0);
+
+    // Create one tx
+    const tx1 = await wallet.send({
+      outputs: [{
+        address: recAddr,
+        value: 10000}]
+    });
+    ancs = await wallet.getPendingAncestors(tx1);
+    assert.strictEqual(ancs.size, 1);
+
+    // Create a second tx
+    const tx2 = await wallet.send({
+      outputs: [{
+        address: recAddr,
+        value: 10000}]
```

Comment:
Is this formatting common? I almost thought there was an indentation problem or missing brackets.

Reply:
ok fixed in both new tests and rebased